### PR TITLE
Removed unfinished features from UI

### LINKED ISSUE #380: Hide unfinished features
As a user I don't want to see any buttons, texts and components that are not implemented or are related to features that are unfinished. This would help my work by clearing the UI from clutter.

### PR DESCRIPTION
**Pull request by Kaemppi**
resolves #380

### CODE REVIEW

File: src/components/generic/card/card-list.tsx [hunk 1]
```diff
@@ -38,11 +45,16 @@ class CardList extends React.Component<Props, State> {
...
     return (
-      <div className={ classes.cardView }>
-        <Typography style={{ marginBottom: theme.spacing(2) }} variant="h2">{ title }</Typography>
+      <div className={ autoHeight ? classes.cardViewAutoHeight : classes.cardView }>
+        { title &&
+          <Typography style={ subtitle ? { marginBottom: theme.spacing(1) } : { marginBottom: theme.spacing(2) }} variant="h2">{ title }</Typography>
+        }
+        { subtitle &&
+          <Typography style={{ marginBottom: theme.spacing(2) }} variant="body1">{ subtitle }</Typography>
+        }
```
Comment (lauriLukkarinen) [inline src/components/generic/card/card-list.tsx:57]:
```suggestion
      <div className={ autoHeight ? classes.cardViewAutoHeight : classes.cardView }>
        { title &&
          <Typography
            style={{ marginBottom: theme.spacing(subtitle ? 1 : 2) }}
            variant="h2"
          >
            { title }
          </Typography>
        }
        { subtitle &&
          <Typography
            style={{ marginBottom: theme.spacing(2) }}
            variant="body1"
          >
            { subtitle }
          </Typography>
        }
```

File: src/components/screens/group-content-versions-screen.tsx [hunk 2]
```diff
@@ -121,7 +121,7 @@ class GroupContentVersionsScreen extends React.Component<Props, State> {
         { this.renderGroupContentVersionCardsList() }
         { this.renderAddDialog() }
         { this.renderConfirmDeleteDialog() }
-        <ElementSettingsPane open={ true } title="Properties" width={ selectedGroupContentVersion && contentVersion ? 320 : 0 }>
+        <ElementSettingsPane open={ true } title={ strings.generic.properties } width={ selectedGroupContentVersion && contentVersion ? 320 : 0 }>
```
Comment (lauriLukkarinen) [inline src/components/screens/group-content-versions-screen.tsx:124]:
```suggestion
        <ElementSettingsPane
          open={ true }
          title={ strings.generic.properties }
          width={ selectedGroupContentVersion && contentVersion ? 320 : 0 }
        >
```

File: src/components/screens/layouts-screen.tsx [hunk 3]
```diff
@@ -148,21 +147,20 @@ class LayoutsScreen extends React.Component<Props, State> {
       return (
         <CardItem
           key={ subLayout.id }
-          title={ `${strings.subLayout.title} - ${subLayout.name}` }
+          title={ `${ strings.subLayout.name } - ${ subLayout.name }` }
```
Comment (lauriLukkarinen) [inline src/components/screens/layouts-screen.tsx:150]: String literals are written without spaces around the variables within string
```suggestion
          title={ `${strings.subLayout.name} - ${subLayout.name}` }
```

File: src/components/screens/layouts-screen.tsx [hunk 4]
```diff
@@ -148,21 +147,20 @@ class LayoutsScreen extends React.Component<Props, State> {
...
 
     return (
       <div style={{ width: "100%", overflowY: "auto" }}>
-        <CardList title={ strings.layout.title }>
+        <CardList autoHeight>
           { layoutCards }
         </CardList>
 
-        <CardList title={ strings.subLayout.title }>
+        <CardList autoHeight title={ strings.subLayout.title } subtitle={ strings.subLayout.description }>
```
Comment (lauriLukkarinen) [inline src/components/screens/layouts-screen.tsx:163]:
```suggestion
        <CardList
          autoHeight
          title={ strings.subLayout.title }
          subtitle={ strings.subLayout.description }
        >
```

File: src/components/screens/layouts-screen.tsx [hunk 5]
```diff
@@ -224,15 +222,17 @@ class LayoutsScreen extends React.Component<Props, State> {
     const { newLayout } = this.state;
 
     return (
-      <>
+      <FormControl
+        variant="outlined"
+      >
```
Comment (lauriLukkarinen) [inline src/components/screens/layouts-screen.tsx:227]:
```suggestion
      <FormControl variant="outlined">
```

**Review by lauriLukkarinen — CHANGES_REQUESTED** [review]

File: src/components/generic/card/card-list.tsx (hunk 1 above)
Comment (Kaemppi) [reply src/components/generic/card/card-list.tsx]: ah paljon nätinpi

**Review by lauriLukkarinen — APPROVED** [review]

**Pull request merged**